rdmo/core/management/commands/find_spam_users.py

```diff
@@ -15,8 +15,8 @@ def add_arguments(self, parser):
             ', default is 2'
         )
         parser.add_argument(
-            '-n', '--occurence', default=3, type=int,
-            help='number of sequentially occuring timespan ' +
+            '-n', '--occurrence', default=3, type=int,
```

Comment:
This is an API change, but the effect is minor in my opinion. Should this be fixed or left as is?

Reply:
Thanks, this can be fixed.